spark/src/main/java/com/netflix/iceberg/spark/source/Writer.java

```diff
@@ -190,10 +197,13 @@ public String toString() {
       AppenderFactory<InternalRow> factory = new SparkAppenderFactory();
       if (spec.fields().isEmpty()) {
         return new UnpartitionedWriter(
-            fileIo.newOutputFile(locations.newDataLocation(filename)), format, factory, fileIo);
+            fileIo.newOutputFile(locations.newDataLocation(filename)), format, factory, fileIo, encryptionManager);
```

Comment:
Why pass the encryption manager to `UnpartitionedWriter` but not to `PartitionedWriter`? To keep them at the same abstraction, I think this should pass `EncryptedOutputFile`.

Reply:
@yifeih, did you intend to update this as well?

Reply:
oh hmm i thought i did already, but i guess i remembered wrong. i'll make that change right now!

Reply:
Oh i misunderstood this, I did the opposite... ok i'll fix this

Reply:
Sorry! I didn't check for that option. Either way is fine with me.